uptane-standard.md

```diff
@@ -860,16 +860,16 @@ In order to perform full verification, an ECU SHALL perform the following steps:
 8. Download and check the Timestamp metadata file from the Image repository, following the procedure in {{check_timestamp}}.
 9. Download and check the Snapshot metadata file from the Image repository, following the procedure in {{check_snapshot}}.
 10. Download and check the top-level Targets metadata file from the Image repository, following the procedure in {{check_targets}}.
-11. For each image listed in the Targets metadata file from the Director repository downloaded in step 6, complete the following procedure:
-    1. Locate and download a Targets metadata file that contains an image with exactly the same file name, following the procedure in {{resolve_delegations}}.
+11. Verify that Targets metadata from the Director and Image repositories match. A primary ECU MUST perform this check on metadata for all images listed in the Targets metadata file from the Director repository downloaded in step 6. A secondary ECU MAY elect to perform this check only on the metadata for the image it will install. (That is, the metadata from the Director that contains the ECU identifier of the current ECU.) To check that the metadata for an image matches, complete the following procedure:
```

Comment:
Looks fine to me except that the parenthetical text is a sentence fragment, which threw me off for a minute.